### PR TITLE
select: Check jsdoc

### LINKED ISSUE #837: select: Check jsdoc

### PR DESCRIPTION
**Pull request by lapinlapin**
Close #837

### CODE REVIEW

File: common.blocks/select/select.js [hunk 1]
```diff
@@ -119,10 +119,18 @@ provide(BEMDOM.decl(this.name, /** @lends select.prototype */{
...
+     * Get name
+     * @returns {*}
+     */
     getName : function() {
         return this.params.name;
     },
 
+    /**
+     * Get default parameters
+     * @returns {{optionsMaxHeight: Number}}
```
Comment (dfilatov) [inline common.blocks/select/select.js]: `jsd` doesn't support this syntax

**Comment by sipayRT** [comment]
@dfilatov check pls

File: common.blocks/select/select.js [hunk 2]
```diff
@@ -103,7 +103,7 @@ provide(BEMDOM.decl(this.name, /** @lends select.prototype */{
 
     /**
      * Get value
-     * @returns {*}
+     * @returns {String}
```
Comment (dfilatov) [inline common.blocks/select/select.js]: Why?

File: common.blocks/select/select.js [hunk 3]
```diff
@@ -119,6 +119,10 @@ provide(BEMDOM.decl(this.name, /** @lends select.prototype */{
         return this;
     },
 
+    /**
+     * Get name
+     * @returns {*}
```
Comment (dfilatov) [inline common.blocks/select/select.js]: name is always `String`

**Comment by sipayRT** [comment]
@lapinlapin

**Comment by lapinlapin** [comment]
@sipayRT @dfilatov

**Comment by dfilatov** [comment]
lgtm

**Pull request merged**